### PR TITLE
Migrate Emoji.GetMultipleByName to Sync by default

### PR DESCRIPTION
**Pull request by gupsho**
#### Summary
Migrate "Emoji.GetMultipleByName" to Sync by default

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/11295

### CODE REVIEW

File: store/sqlstore/emoji_store.go [hunk 1]
```diff
@@ -122,25 +122,22 @@ func (es SqlEmojiStore) GetByName(name string) store.StoreChannel {
...
+	keys, params := MapStringsToQueryParams(names, "Emoji")
 
-		var emojis []*model.Emoji
+	var emojis []*model.Emoji
 
-		if _, err := es.GetReplica().Select(&emojis,
-			`SELECT
+	if _, err := es.GetReplica().Select(&emojis,
+		`SELECT
 				*
```
Comment (jespino) [inline store/sqlstore/emoji_store.go]: Please keep the indentation consistent here.

Comment (gupsho) [reply store/sqlstore/emoji_store.go]: Done.

**Comment by gupsho** [comment]
> Just a tiny detail about indentation, but otherwise looks good to me. Thanks @gupsho! 🕺

Fixed.

**Comment by jespino** [comment]
@gupsho please avoid the `push -f` when the PRs are already on review, because now I have to review again the whole thing instead of relay on what you changed. Not a big deal with this size of PR, but anyway a good policy to follow :)

**Comment by gupsho** [comment]
> @gupsho please avoid the `push -f` when the PRs are already on review, because now I have to review again the whole thing instead of relay on what you changed. Not a big deal with this size of PR, but anyway a good policy to follow :)

How do I rebase and push then?

**Comment by jespino** [comment]
@gupsho good question, the idea is that you merge master into your branch instead of reading. Then when the PR is ready we always "squash and merge" the PR so it ends up being just one commit without merges so the final result is exactly the same.

**Comment by cpanato** [comment]
@gupsho can you please update your PR? thanks!

**Comment by cpanato** [comment]
@gupsho please update your PR and resolve the conflict, thanks!

**Pull request merged**